.github/workflows/build-container.yaml

```
@@ -0,0 +1,37 @@
+      - ojin-pre-release
+
+jobs:
+  build-test:
+    name: Test Container Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.build_from_branch || github.ref }}
```

actionlint (1.7.8)

property "build_from_branch" is not defined in object type {}

(expression)